src/csharp/Microsoft.Spark.Worker/Microsoft.Spark.Worker.csproj

```diff
@@ -6,6 +6,7 @@
     <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netcoreapp3.1</TargetFrameworks>
     <RootNamespace>Microsoft.Spark.Worker</RootNamespace>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <RestoreAdditionalProjectSources>https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet3.1/nuget/v3/index.json</RestoreAdditionalProjectSources>
```

Comment:
Indentation looks off?

Reply:
Can you also add a comment which package is being downloaded from this source?

Reply:
I think indentation looks off because it's a long line.  Added a comment, however in the future we should be able to remove this additional repo once this package gets published to nuget.org.